src/dialects/mysql/query-generator.js

```diff
@@ -628,18 +639,6 @@ export class MySqlQueryGenerator extends AbstractQueryGenerator {
 
     return `json_unquote(json_extract(${quotedColumn},${pathStr}))`;
   }
-
-  _createBindParamCollector(bindContext /* : BindContext */) {
```

Comment:
At first I was wondering why you removed this here as it doesn't relate to the scope of this PR, but I don't see it being used so good that we remove it

Reply:
It looks like a remnant of the "fix sql injection in replacements" PR that never got removed

Reply:
Yeah indeed